src/test/java/com/thoughtworks/pacman/ui/BackgroundSoundLoaderTest.java

```diff
@@ -0,0 +1,34 @@
+package com.thoughtworks.pacman.ui;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.mock;
+
+@RunWith(MockitoJUnitRunner.class)
+
+public class BackgroundSoundLoaderTest{
+    private BackgroundSoundLoader backgroundSoundLoader;
+
+    @Test (expected = ThreadDeath.class)
+    public void setUp() throws Exception {
+        backgroundSoundLoader = new BackgroundSoundLoader();
+        backgroundSoundLoader.run();
+        assertTrue(throwException());
```

Comment:
Testing isminden neyi test ettiğin anlaşılmıyor. Burada niye exception atılmasını bekliyoruz onu da anlamadım.